src/component/term/TermMetadataEdit.tsx

```diff
@@ -85,18 +85,11 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
         this.setState({scopeNote: Object.assign({}, this.state.scopeNote, change)});
     };
 
-    public onDefinitionChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        const value = e.currentTarget.value;
-        const change = {};
-        change[this.props.language] = value;
-        this.setState({definition: Object.assign({}, this.state.definition, change)});
+    public onChange = (change: Partial<TermData>) => {
+        // @ts-ignore
```

Comment:
Why this?

Reply:
Won't compile without the ignore. Partial marks all properties optional, but label is not in the state.